lib/Mojo/Date.pm

```diff
@@ -69,6 +71,16 @@ sub to_string {
     $MONTHS[$month], $year + 1900, $h, $m, $s;
 }
 
+# ($t, $f) = _modf($x)
```

Comment:
I updated the PR with the shortened version of _modf(). 

(Also changed the output prototype to match the one of POSIX::modf() – which unfortunately is not what I needed here – gives |$f| < 1 rather than 0 ≤ $f 1.)